gybitg/Migrations/20190531010057_GalleryVideos model update.cs

```diff
@@ -0,0 +1,85 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace gybitg.Migrations
+{
+    public partial class GalleryVideosmodelupdate : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<string>(
+                name: "GalleryVideo1",
```

Comment:
Brenda, listing out individual GalleryVideo columns in the data tables will be a problem as the application grows; where you will have to change the database structure to add a new column. Instead, create a many-to-many relationship that can continue to add as many records as needed.

Reply:
@Bansenauer-Cascadia ah yes that makes sense. :)

Reply:
I will see how far I can get in refactoring this tonight with the last of my hours. :)